sql/data_collection_helpers/bloated_tables_and_indexes.sql

```diff
@@ -123,16 +123,23 @@ DECLARE
       bloat_size::numeric,
       pg_size_pretty(bloat_size::int8)
     FROM (
-        SELECT current_database(), nspname AS schemaname, tblname, idxname, bs*(sub.relpages)::bigint AS real_size,
-        bs*est_pages::bigint as estimated_size,
-        bs*(sub.relpages-est_pages)::bigint AS bloat_size,
-        100 * (sub.relpages-est_pages)::float / sub.relpages AS bloat_ratio, is_na
+        SELECT current_database(), nspname AS schemaname, tblname, idxname, bs*(relpages-est_pages)::bigint AS extra_size,
+        100 * (relpages-est_pages)::float / relpages AS extra_ratio,
+        fillfactor, bs*(relpages-est_pages_ff) AS bloat_size,
```

Comment:
We may opt to show those additional columns in new version of PGO. Besides, I wanted to preserve the original query as much as possible (I only stripped the comments). It will make further changes to our version easier to make if the original changes.